From ce1cdc1e2c74e58bd83374163453e51c1c32e927 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Mon, 4 Aug 2014 08:13:42 -0400
Subject: [redhat] kernel.spec: remove accidental duplicate clog

Kernel 142 should have been 141, but thanks to breakage in rhpkg and it
being late on a Friday afternoon/evening when it was committed and sent to
brew, the duplicated runs of make rh-release and make rh-release-tag
weren't caught until it was too late. No harm in it though, other than the
duplicated changelog, which I'm tweaking right here... Now, it would be
nice if someone would fix rhpkg...

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 04809ee..d8745d0 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1581,113 +1581,7 @@ fi
 - [virt] xen-blkfront: handle backend CLOSED without CLOSING (Vitaly Kuznetsov) [1070868]
 
 * Fri Aug 01 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-141.el7]
-- [pci] Don't check resource_size() in pci_bus_alloc_resource() (Myron Stowe) [1110895]
-- [pci] Enable INTx in pci_reenable_device() only when MSI/MSI-X not enabled (Myron Stowe) [1110895]
-- [ahci] Fix broken fallback to single MSI mode (Myron Stowe) [1110895]
-- [pci] Enable INTx if BIOS left them disabled (Myron Stowe) [1110895]
-- [pci] msi: Add pci_enable_msi_exact() and pci_enable_msix_exact() (Myron Stowe) [1110895]
-- [pci] msi: Fix cut-and-paste errors in documentation (Myron Stowe) [1110895]
-- [pci] msi: Add pci_enable_msi() documentation back (Myron Stowe) [1110895]
-- [pci] msi: Fix pci_msix_vec_count() htmldocs failure (Myron Stowe) [1110895]
-- [pci] msi: Fix leak of msi_attrs (Myron Stowe) [1110895]
-- [pci] msi: Check kmalloc() return value, fix leak of name (Myron Stowe) [1110895]
-- [pci] Check parent kobject in pci_destroy_dev() (Myron Stowe) [1110895]
-- [powerpc] eeh: Use global PCI rescan-remove locking (Myron Stowe) [1110895]
-- [pci] Fix pci_check_and_unmask_intx() comment typos (Myron Stowe) [1110895]
-- [scsi] mpt: Use pci_stop_and_remove_bus_device_locked() (Myron Stowe) [1110895]
-- [platform] x86: Use global PCI rescan-remove locking (Myron Stowe) [1110895]
-- [pci] hotplug: Use global PCI rescan-remove locking (Myron Stowe) [1110895]
-- [pcmcia] Use global PCI rescan-remove locking (Myron Stowe) [1110895]
-- [pci] hotplug: Use global PCI rescan-remove locking (Myron Stowe) [1110895]
-- [acpi] pci: Use global PCI rescan-remove locking in PCI root hotplug (Myron Stowe) [1110895]
-- [pci] Add global pci_lock_rescan_remove() (Myron Stowe) [1110895]
-- [pci] Cleanup pci.h whitespace (Myron Stowe) [1110895]
-- [pci] Reorder so actual code comes before stubs (Myron Stowe) [1110895]
-- [pci] aer: Support ACPI HEST AER error sources for PCI domains other than 0 (Myron Stowe) [1110895]
-- [acpi] acpica: Add helper macros to extract bus/segment numbers from HEST table (Myron Stowe) [1110895]
-- [pci] Make local functions static (Myron Stowe) [1110895]
-- [pci] Remove unused alloc_pci_dev() (Myron Stowe) [1110895]
-- [pci] Remove unused pci_renumber_slot() (Myron Stowe) [1110895]
-- [pci] Remove unused pcie_aspm_enabled() (Myron Stowe) [1110895]
-- [pci] Remove unused pci_vpd_truncate() (Myron Stowe) [1110895]
-- [pci] Remove unused ID-Based Ordering support (Myron Stowe) [1110895]
-- [pci] Remove unused Optimized Buffer Flush/Fill support (Myron Stowe) [1110895]
-- [pci] Remove unused Latency Tolerance Reporting support (Myron Stowe) [1110895]
-- [pci] Removed unused parts of Page Request Interface support (Myron Stowe) [1110895]
-- [pci] Update documentation 00-INDEX file (Myron Stowe) [1110895]
-- [pci] Allocate 64-bit BARs above 4G when possible (Myron Stowe) [1110895]
-- [pci] Enforce bus address limits in resource allocation (Myron Stowe) [1110895]
-- [pci] Split out bridge window override of minimum allocation address (Myron Stowe) [1110895]
-- [agp] ati: Use PCI_COMMAND instead of hard-coded 4 (Myron Stowe) [1110895]
-- [agp] intel: Use CPU physical address, not bus address, for ioremap() (Myron Stowe) [1110895]
-- [agp] intel: Use pci_bus_address() to get GTTADR bus address (Myron Stowe) [1110895]
-- [agp] intel: Use pci_bus_address() to get MMADR bus address (Myron Stowe) [1110895]
-- [agp] intel: Support 64-bit GMADR (Myron Stowe) [1110895]
-- [agp] intel: Rename gtt_bus_addr to gtt_phys_addr (Myron Stowe) [1110895]
-- [drm] i915: Rename gtt_bus_addr to gtt_phys_addr (Myron Stowe) [1110895]
-- [agp] Use pci_resource_start() to get CPU physical address for BAR (Myron Stowe) [1110895]
-- [agp] Support 64-bit APBASE (Myron Stowe) [1110895]
-- [pci] msi: Add pci_enable_msi_range() and pci_enable_msix_range() (Myron Stowe) [1110895]
-- [pci] msi: Add pci_msix_vec_count() (Myron Stowe) [1110895]
-- [pci] msi: Remove pci_enable_msi_block_auto() (Myron Stowe) [1110895]
-- [pci] msi: Add pci_msi_vec_count() (Myron Stowe) [1110895]
-- [pci] Convert ioapic to be builtin only, not modular (Myron Stowe) [1110895]
-- [pci] Add pci_bus_address() to get bus address of a BAR (Myron Stowe) [1110895]
-- [pci] Convert pcibios_resource_to_bus() to take a pci_bus, not a pci_dev (Myron Stowe) [1110895]
-- [pci] Change pci_bus_region addresses to dma_addr_t (Myron Stowe) [1110895]
-- [pci] msi: Return -ENOSYS for unimplemented interfaces, not -1 (Myron Stowe) [1110895]
-- [pci] msi: Return msix_capability_init() failure if populate_msi_sysfs() fails (Myron Stowe) [1110895]
-- [s390] pci: Remove superfluous check of MSI type (Myron Stowe) [1110895]
-- [s390] pci: Fix single MSI only check (Myron Stowe) [1110895]
-- [pci] msi: Export MSI mode using attributes, not kobjects (Myron Stowe) [1110895]
-- [pci] portdrv: Remove extra get_device()/put_device() for pcie_device (Myron Stowe) [1110895]
-- [pci] portdrv: Add put_device() after device_register() failure (Myron Stowe) [1110895]
-- [pci] portdrv: Cleanup error paths (Myron Stowe) [1110895]
-- [pci] Remove from bus_list and release resources in pci_release_dev() (Myron Stowe) [1110895]
-- [pci] Move pci_proc_attach_device() to pci_bus_add_device() (Myron Stowe) [1110895]
-- [pci] Use device_release_driver() in pci_stop_root_bus() (Myron Stowe) [1110895]
-- [pci] aer: Consolidate HEST error source parsers (Myron Stowe) [1110895]
-- [pci] aer: Ignore non-PCIe AER error sources in aer_hest_parse() (Myron Stowe) [1110895]
-- [pci] aer: Clean up error printing code a bit (Myron Stowe) [1110895]
-- [pci] aer: Add a TLP header print helper (Myron Stowe) [1110895]
-- [eisa] Call put_device() if device_register() fails (Myron Stowe) [1110895]
-- [pci] checkpatch: Deprecate DEFINE_PCI_DEVICE_TABLE (Myron Stowe) [1110895]
-- [pci] Stop clearing bridge Secondary Status when setting up I/O aperture (Myron Stowe) [1110895]
-- [pci] Prevent bus conflicts while checking for bridge apertures (Myron Stowe) [1110895]
-- [pci] Drop "irq" param from *_restore_msi_irqs() (Myron Stowe) [1110895]
-- [pci] portdrv: Remove superfluous name cast (Myron Stowe) [1110895]
-- [x86] pci: Use dev_is_pci() to identify PCI devices (Myron Stowe) [1110895]
-- [pci] Use dev_is_pci() to identify PCI devices (Myron Stowe) [1110895]
-- [pci] Clear NumVFs when disabling SR-IOV in sriov_init() (Myron Stowe) [1110895]
-- [s390] watchdog: add support for LPAR operation (diag288) (Hendrik Brueckner) [1088540]
-- [s390] watchdog: use watchdog API (Hendrik Brueckner) [1088540]
-- [scripts] headers_check: special case seqbuf_dump() (Prarit Bhargava) [1090078]
-- [Documentation] mei-amt-version: remove unneeded call of mei_deinit() (Prarit Bhargava) [1090078]
-- [drm] tegra: Include header drm/drm.h (Prarit Bhargava) [1090078]
-- [mfd] lpc_ich: iTCO_wdt patch for Intel Coleto Creek DeviceIDs (Neil Horman) [1053912]
-- [i2c] i801: SMBus patch for Intel Coleto Creek DeviceIDs (Neil Horman) [1053912]
-- [x86] efi: earlyprintk-efi, keep fix (Rui Wang) [1085558]
-- [x86] efi: Fix earlyprintk off-by-one bug (Rui Wang) [1085558]
-- [x86] efi: Add EFI framebuffer earlyprintk support (Rui Wang) [1085558]
-- [s390] zcrypt: remove zcrypt kmsg documentation again (Hendrik Brueckner) [1088511]
-- [s390] zcrypt: add support for EP11 coprocessor cards (Hendrik Brueckner) [1088511]
-- [s390] sclp_vt220: Enable ASCII console per default (Hendrik Brueckner) [1112739]
-- [block] blkcg: fix use-after-free in __blkg_release_rcu() by making blkcg_gq refcnt an atomic_t (Vivek Goyal) [1118436]
-- [ethernet] bnx2x: fix crash during TSO tunneling (Michal Schmidt) [1119133]
-- [ethernet] cxgb4: allow large buffer size to have page size (Gustavo Duarte) [1078977]
-- [fs] vfs: Don't copy mount bind mounts of /proc/<pid>/ns/mnt between namespaces (Aristeu Rozanski) [1098632]
-- [cpufreq] intel_pstate: Add CPU IDs for Broadwell processors (Steve Best) [1086874]
-- [pci] pciehp: Clear Data Link Layer State Changed during init (Myron Stowe) [1101097]
-- [acpi] Fix bug when ACPI reset register is implemented in system memory (Nigel Croxon) [1109971]
-- [ipc] shm: increase the defaults for SHMALL, SHMMAX (Larry Woodman) [1101821]
-- [tools] power/turbostat: Run on Broadwell (Steve Best) [1086455]
-- [drm] i915: make i915 less chatty on resume (Rob Clark) [1086777]
-- [tools] cpupower: add option to display values without round offs (Prarit Bhargava) [1090540]
-- [scripts] kbuild: fix make headers_install when path is too long (Frank Ramsay) [1099575]
-- [kernel] audit: fix AUDIT_FEATURE_CHANGE record number (Richard Guy Briggs) [1085928]
-- [usb] xhci: Link TRB must not occur within a USB payload burst (Don Zickus) [1049077]
-- [virt] xen-netfront: fix resource leak in netfront (Vitaly Kuznetsov) [1070868]
-- [virt] xen-blkfront: handle backend CLOSED without CLOSING (Vitaly Kuznetsov) [1070868]
+- [misc] no animals were harmed in the making of this kernel (Jarod Wilson)
 
 * Fri Jul 25 2014 Jarod Wilson <jarod@redhat.com> [3.10.0-140.el7]
 - [net] ipv6: addrconf: implement address generation modes (Jiri Pirko) [1107369]
-- 
1.7.1